Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
+
+  # lots of OS X support things were missing from this release
+  patch :DATA
```

Comment:
@n3npq Homebrew is unusual compared to other package managers in that we want to avoid patching whenever possible. In this case we have an existing version that works without patching so we'd rather work with upstream to reach a process such that versions that will be included into Homebrew work unpatched on OS X before we include them.